src/acl/acl.ts

```diff
@@ -142,20 +142,17 @@ export function internal_getContainerPath(resourcePath: string): string {
 }
 
 /**
- * Verify whether an ACL was found for the given Resource.
+ * Verify whether the given Resource has a resource ACL attached.
  *
- * A Resource fetched with its ACL (e.g. using [[getSolidDatasetWithAcl]]) _might_ have a resource ACL attached, but
- * we cannot be sure: it might be that none exists for this specific Resource (in which case the
- * fallback ACL applies), or the currently authenticated user (if any) might not have Control access
- * to the fetched Resource.
+ * A Resource returned using [[getSolidDatasetWithAcl]], i.e., the returned `dataset`,
+ * has its resource ACL attached if its resource ACL exists and the current user
+ * has Control access to the Resource. Otherwise, the Resource will not have a resource ACL attached.
  *
- * This function verifies that the Resource's ACL is accessible.
- *
- * Please note that the Web Access Control specification is not yet finalised, and hence, this
- * function is still experimental and can change in a non-major release.
+ * _Note_: The Web Access Control specification is not yet finalised. As such, this
```

Comment:
If you don't like it as a note, can revert.  Just wasn't sure how to draw attention to it.

Reply:
No objection to using `_Note_:` from my side.